### PR TITLE
ref(py3): low-hanging fruit part 2

### PR DESCRIPTION
**Pull request by joshuarli**
Follow up to https://github.com/getsentry/sentry/pull/17288. From `663 failed, 4967 passed` to `471 failed, 5160 passed`. The goal is to just fix as much of the easy and most verbose stuff as possible, before thinking about applying more mass-refactors, so test regressions are more clear.

Regarding bytes vs. unicode changes in here, they are the most obvious ones. There are plenty more, but fixing those involves deciding whether our APIs are expected to return bytes vs unicode, so I don't consider those low-hanging fruit. For example:

```
  File "/home/travis/build/getsentry/sentry/src/sentry/api/serializers/rest_framework/origin.py", line 26, in is_valid_origin
    if ":" in bits.domain:
TypeError: a bytes-like object is required, not 'str'
```

`binary_type` and `bytes.encode` audit will be coming up next.

### CODE REVIEW

File: src/sentry/testutils/cases.py [hunk 1]
```diff
@@ -498,7 +498,7 @@ def __helper_enable_organization_2fa(self, organization, user, status_code, err_
         response = self.api_enable_org_2fa(organization, user)
         assert response.status_code == status_code
         if err_msg:
-            assert err_msg in response.content
+            assert err_msg.encode("utf-8") in response.content
```
Comment (wedamija) [inline src/sentry/testutils/cases.py]: Is this correct? Or are we masking an error in that `response.content` has changed?

File: src/sentry/testutils/cases.py [hunk 2]
```diff
@@ -745,7 +745,7 @@ def setUp(self):
         self.save_session()
 
     def assertDialogSuccess(self, resp):
-        assert "window.opener.postMessage(" in resp.content
+        assert b"window.opener.postMessage(" in resp.content
```
Comment (wedamija) [inline src/sentry/testutils/cases.py]: Just checking that it makes sense for this to be bytes here.

**Review by wedamija — APPROVED** [review]

File: src/sentry/testutils/cases.py (hunk 1 above)
Comment (joshuarli) [reply src/sentry/testutils/cases.py]: django `HttpResponse.content` is always a bytestring. https://docs.djangoproject.com/en/1.11/ref/request-response/#django.http.HttpResponse.content

File: src/sentry/testutils/cases.py (hunk 2 above)
Comment (joshuarli) [reply src/sentry/testutils/cases.py]: https://github.com/getsentry/sentry/pull/17382#discussion_r385986979

**Pull request merged**